package.json

```diff
@@ -115,19 +112,16 @@
     "arrowParens": "avoid",
     "singleQuote": true
   },
-  "resolutions": {
-    "package-build-stats/webpack": "pastelsky/webpack-4-modern#fca4748eafe351bf9757c04ef9fcce2151c6ac14"
-  },
   "devDependencies": {
     "@babel/core": "^7.17.10",
     "@babel/preset-env": "^7.17.10",
-    "@zeit/next-source-maps": "^0.0.4-canary.1",
     "autoprefixer": "^9.8.6",
     "babel-jest": "^24.9.0",
     "babel-plugin-css-modules-transform": "^1.6.1",
     "babel-plugin-inline-react-svg": "^1.1.1",
     "babel-plugin-module-resolver": "^4.0.0",
     "babel-plugin-wrap-in-js": "^1.1.1",
+    "eslint-config-next": "^13.0.0",
```

Comment:
Why has this been added?

Reply:
I believe it's was a postinstall script that installed this. I can try remove it see if build fails.

Reply:
Removed this package